src/app/(headerless)/login/hooks/useLogin.ts

```diff
@@ -0,0 +1,11 @@
+import { useAuth } from '@/hooks/useAuth';
+import { useRouter } from 'next/navigation';
+import { useEffect } from 'react';
+
+export default function useLogin() {
+  const { auth } = useAuth();
+  const router = useRouter();
+  useEffect(() => {
+    if (auth !== null) router.push('/home');
```

Comment:
이 useLogin 훅은 로그인 된 유저가 로그인 페이지에 접근했을 때, `/home`으로 이동시켜주는 훅 같은데, 다시 뒤로가기 했을 때 로그인 페이지로 접근하지 못하도록 하는 게 좋을 것 같아서 `router.push` 보다는 `router.replace`를 사용하는 게 좋지 않을까 생각합니닷 !!

```suggestion
export default function useLogin() {
  const { auth } = useAuth();
  const router = useRouter();
  useEffect(() => {
    if (auth !== null) router.replace('/home');
```